### ISSUE TITLE
xlog error on first box.cfg call

### ISSUE THREAD

**Issue by olegrok** [tarantool/tarantool#4510]
```
➜  dec-test ls
➜  dec-test tarantool
Tarantool Enterprise For Mac 2.3.0-10-g465c23ffe
type 'help' for interactive help
tarantool> box.cfg{}
'strip_core' is set but unsupported
2019-09-18 15:31:57.129 [41475] main/102/interactive C> Tarantool Enterprise For Mac 2.3.0-10-g465c23ffe
2019-09-18 15:31:57.129 [41475] main/102/interactive C> log level 5
2019-09-18 15:31:57.129 [41475] main/102/interactive I> mapping 268435456 bytes for memtx tuple arena...
2019-09-18 15:31:57.130 [41475] main/102/interactive I> mapping 134217728 bytes for vinyl tuple arena...
2019-09-18 15:31:57.131 [41475] main/102/interactive I> instance uuid d01b6b5f-5743-4a4c-83c5-bff5280235f7
2019-09-18 15:31:57.131 [41475] main/102/interactive I> initializing an empty data directory
-- Here:
2019-09-18 15:31:57.140 [41475] main/102/interactive xlog.c:1934 E> can't open tx: bootstrap: has some data after eof marker at 5902

2019-09-18 15:31:57.140 [41475] main/102/interactive I> assigned id 1 to replica d01b6b5f-5743-4a4c-83c5-bff5280235f7
2019-09-18 15:31:57.140 [41475] main/102/interactive I> cluster uuid c414543a-3e53-4ec9-8686-44510262b4fb
2019-09-18 15:31:57.141 [41475] snapshot/101/main I> saving snapshot `./00000000000000000000.snap.inprogress'
2019-09-18 15:31:57.142 [41475] snapshot/101/main I> done
2019-09-18 15:31:57.142 [41475] main/102/interactive I> ready to accept requests
2019-09-18 15:31:57.142 [41475] main/104/checkpoint_daemon I> scheduled next checkpoint for Wed Sep 18 17:23:42 2019
```

**Comment by kyukhin** [comment]
Do you have a reproducer?

**Comment by olegrok** [comment]
Can't reproduce on Tarantool community master, but reproduced on Enterprise versions (on MacOS and Linux)

**Comment by rosik** [comment]
Tarantool 2.2.1-0-g4138645e0

Reproducer:

```console
> box.cfg()
...
... xlog.c:1934 E> can't open tx: bootstrap: has some data after eof marker at 5902
...
```

See also #3460

**Comment by rosik** [comment]
Tarantool 1.10.4-33-g939def455 seems to be OK.

**Comment by kyukhin** [comment]
Fixed by 856662f
